src/sentry/static/sentry/app/components/events/interfaces/debugMeta-v2/index.tsx

```diff
@@ -499,13 +511,24 @@ class DebugMeta extends React.PureComponent<Props, State> {
   }
 
   render() {
+    const {data} = this.props;
+    const {images} = data;
+
     const {
       searchTerm,
       filterOptions,
       scrollbarWidth,
-      filteredImagesByFilter: images,
+      filteredImagesByFilter: filteredImages,
     } = this.state;
 
+    if (
+      !filteredImages.length &&
+      (images.every(image => image?.type === 'proguard') ||
+        images.every(image => !defined(image)))
```

Comment:
Should we do both checks in one go so that we don't have to iterate over images two times?

Reply:
I updated the code with a more readable code :)